src/main/scss/pages/_about.scss

```diff
@@ -119,18 +119,18 @@
 .app-about-heading {
   font-weight: 600;
   font-family: Georgia, serif;
-  font-size: 1.6rem;
+  font-size: 1.5rem;
```

Comment:
Not really related - but `1.6rem` is `25.6px` which would look odd as its not a whole number.